.github/ISSUE_TEMPLATE/new-release-template.md

```diff
@@ -7,30 +7,41 @@ assignees: comcmaho, amdeel, davidmrdavid, bachuv
 
-
-**Validation (assigned to: )**
+**Prep Release (assigned to: )**
 _Due: <2-business-days-before-release>_
-- [ ] Run private performance tests and ensure no regressions
-- [ ] Smoke test Functions V1 and Functions V3 .NET apps
-- [ ] Smoke test JavaScript and Python apps
-- [ ] Check for package size, make sure it's not surprisingly heavier than a previous release
+- [ ] Update Durable Functions references (Analyzer? DTFx?) and increment version.
+- [ ] Add Durable Functions package to myget staging feed.
+- [ ] Check for package size, make sure it's not surprisingly heavier than a previous release.
+- [ ] Merge dev into main (Merge NOT Squash merge).
+
+**Validation**
+_Due: <1-business-days-before-release>_
+- [ ] Run private performance tests and ensure no regressions. **(assigned to: )**
+- [ ] Smoke test Functions V1, Functions V2, and Functions V3 .NET apps. **(assigned to: )**
+- [ ] Smoke test JavaScript and Python apps. **(assigned to: )**
 
+**DTFx Release Completion (assigned to: )**
+_Due: <release-deadline>_
+- [ ] Publish DTFx packages to nuget (directly to nuget.org).
+- [ ] Publish release notes for DTFx.
 
-** Release Completion (assigned to: )**:
+**Release Completion (assigned to: )**
 _Due: <release-deadline>_
-- [ ] Push staged package on MyGet to Nuget.org (for Durable Task Framework, you may need to manually update them)
-- [ ] Create a PR in the [Azure Functions templates repo](https://github.com/Azure/azure-functions-templates) to update templates to the latest version
-- [ ] Create a PR in the [Azure Functions bundles repo](https://github.com/Azure/azure-functions-extension-bundles) to update bundles to the latest version
-- [ ] Merge all pending PR docs from `pending_docs.md`
+- [ ] Delete Durable Functions packages from myget.
+- [ ] Run Durable Functions release pipeline.
+- [ ] Push myget package to nuget (nuget.org extensions package option).
+- [ ] Create a PR in the [Azure Functions templates repo](https://github.com/Azure/azure-functions-templates) to update templates to the latest version.
+- [ ] Create a PR in the [Azure Functions bundles repo](https://github.com/Azure/azure-functions-extension-bundles) to update bundles to the latest version.
+- [ ] Merge all pending PR docs from `pending_docs.md.`
 - [ ] Reset `pending_docs.md` and `release_notes.md` in the `dev` branch. You will want to save `release_notes.md` somewhere for when you publish release notes.
-- [ ] Merge `dev` into `main`
-- [ ] Publish release notes from the pre-reset `release_notes.md`
-- [ ] Post announcement on [App Service Announcements GitHub repo](https://github.com/Azure/app-service-announcements) and Twitter.
+- [ ] Merge `dev` into `main.`
+- [ ] Publish release notes from the pre-reset `release_notes.md.`
+- [ ] Post announcement on [App Service Announcements GitHub repo](https://github.com/Azure/app-service-announcements) and Twitter (Chris).
```

Comment:
We haven't really been consistent about using the Announcements GitHub repo. Does anyone have opinions on whether we're adding sufficient value by adding our announcements here? Are there any other announcement avenues we should consider, like internal emails that include CSS, etc.?